examples/gke/manager.yaml

```diff
@@ -182,7 +182,7 @@ spec:
           valueFrom:
```

Comment:
manager.yaml is generated based on https://github.com/scylladb/scylla-operator/blob/978b5fd5f0c2751f97c1a567c960d2b6a84f83d6/config/manager/cluster/cluster.yaml#L73
and 
https://github.com/scylladb/scylla-operator/blob/5688a6fb99b49b2c93913c5195c9eefc30782a3e/config/manager/manager/kustomization.yaml#L9

Reply:
ok - I'll update both.
thanks